src/desktop/apps/artist/components/cta.js

```diff
@@ -1,8 +1,10 @@
+import ArtistPageCTAView from "desktop/components/artist_page_cta/view.coffee"
 import $ from "jquery"
 import metaphysics from "lib/metaphysics.coffee"
 import Artist from "desktop/models/artist.coffee"
+import { once } from "lodash"
 import { data as sd } from "sharify"
-import ArtistPageCTAView from "desktop/components/artist_page_cta/view.coffee"
+const mediator = require("../../../lib/mediator.coffee")
```

Comment:
We can use an absolute path here -- also wanted to check on lines 2 and 3, usually coffee imports need to use `require` rather than `import`.